src/esn.calendar.libs/app/app.constants.js

```diff
@@ -332,6 +332,11 @@
      */
     .constant('CAL_ALARM_MODIFY_COMPARE_KEYS', ['action', 'attendee', 'description', 'summary', 'trigger'])
 
+    /**
+     * When duplicating an event, the JSON keys on the event will be copied.
+     */
+    .constant('CAL_EVENT_DUPLICATE_KEYS', ['attendees', 'title', 'start', 'end', 'location', 'description', 'xOpenpaasVideoconference', 'class'])
```

Comment:
What about resources and alarm?

Reply:
believe it or not, a resource is actually an attendee:
![image](https://user-images.githubusercontent.com/6764881/97424733-6f74dc00-1911-11eb-9ee9-adfbe34e8333.png)